sdk/communication/Azure.Communication.CallingServer/tests/Infrastructure/CallingServerLiveTestBase.cs

```diff
@@ -25,13 +25,13 @@ protected CommunicationIdentityClient CreateInstrumentedCommunicationIdentityCli
                     InstrumentClientOptions(new CommunicationIdentityClientOptions(CommunicationIdentityClientOptions.ServiceVersion.V2021_03_07))));
 
         /// <summary>
-        /// Creates a <see cref="CallClient" />
+        /// Creates a <see cref="CallConnection" />
         /// </summary>
-        /// <returns>The instrumented <see cref="CallClient" />.</returns>
-        protected CallClient CreateInstrumentedCallingServerClient()
+        /// <returns>The instrumented <see cref="CallConnection" />.</returns>
```

Comment:
I think the return should be CallingServerClient

Reply:
This is fixed.